java/arcs/core/crdt/VersionMap.kt

```diff
@@ -99,7 +94,7 @@ class VersionMap(initialData: Map<Actor, Version> = emptyMap()) {
      */
     operator fun minus(other: VersionMap): VersionMap {
```

Comment:
I think the comment for this method could be clearer, it does not return the difference, it returns only increments.

Reply:
Done

Reply:
Only greater-than-zero differences will be returned.

Reply:
Done